packages/rocketchat-webrtc/WebRTCClass.coffee

```diff
@@ -803,9 +803,11 @@ WebRTC = new class
 
 
 Meteor.startup ->
-	RocketChat.Notifications.onUser 'webrtc', (type, data) =>
-		if not data.room? then return
+	Tracker.autorun ->
```

Comment:
Yes, but we can keep like the other code, I need to improve the Notification class to handle this automatically